### PR TITLE
21 may 2024

### PR DESCRIPTION
**Pull request by deltamarnix**

### CODE REVIEW

File: radar.csv [hunk 1]
```diff
@@ -66,17 +70,17 @@ Visual Studio Code,Adopt,tools,FALSE,"<p><strong><a href=""https://code.visualst
...
 Juliahub Secure,Assess,tools,TRUE,"<p><strong><a href=""https://juliahub.com/"">Juliahub Secure</a></strong> is a platform for Julia packages. It is a secure and private registry for Julia packages that can be administered by Deltares. By the end of 2024, they will have vulnerability checks implemented for public packages.</p>"
 Harbor,Assess,tools,TRUE,"<p><strong><a href=""https://goharbor.io/"">Harbor</a></strong> is an open-source cloud-native registry that stores, signs, and scans container images for vulnerabilities. It highly integrates with Kubernetes.</p>"
+pip,Assess,tools,FALSE,"<p><strong><a href=""https://pip.pypa.io/en/stable/"">pip</a></strong> is the package installer for Python. You can use pip to install packages from the Python Package Index and other indexes.</p><p>Motivation: Deltares is using both Conda and Pip, depending on the types of packages. Pip cannot easily distribute binaries as Conda does. Pip is considered slow in dependency resolving. In a lot of project we now use Pixi, which uses uv under the hood to perform quick pip dependency solving.</p>"
```
Comment (joelvdberg) [inline radar.csv:83]:
```suggestion
pip,Assess,tools,FALSE,"<p><strong><a href=""https://pip.pypa.io/en/stable/"">pip</a></strong> is the package installer for Python. You can use pip to install packages from the Python Package Index and other indexes.</p><p>Motivation: Deltares is using both Conda and Pip, depending on the types of packages. Pip cannot easily distribute binaries as Conda does. Pip is considered slow in dependency resolving. In a lot of project we now use Pixi, which uses uv under the hood to perform quick pip dependency resolving.</p>"
```

File: radar.csv [hunk 2]
```diff
@@ -45,17 +46,20 @@ Gitlab on premise,Adopt,platforms,FALSE,"<p><strong><a href=""https://about.gitl
...
+MinIO,Trial,platform,FALSE,"<p><strong><a href=""https://min.io/"">MinIO</a></strong> is a high-performance, distributed object storage server, designed for large-scale private cloud infrastructure.</p>"
+QGIS,Trial,platform,FALSE,"<p><strong><a href=""https://qgis.org/"">QGIS</a></strong> is a free and open-source cross-platform desktop geographic information system (GIS) application that supports viewing, editing, and analysis of geospatial data.</p><p>Motivation: The PMT Hydrology and Hydrodynamics is writing user interface plugins for QGIS and is wondering if these kind of user interfaces is the way to go in the future. If so, we should be able to co-create with QGIS and influence their technical roadmap.</p>"
```
Comment (joelvdberg) [inline radar.csv:57]:
```suggestion
QGIS,Trial,platform,FALSE,"<p><strong><a href=""https://qgis.org/"">QGIS</a></strong> is a free and open-source cross-platform desktop geographic information system (GIS) application that supports viewing, editing, and analysis of geospatial data.</p><p>Motivation: The PMT Hydrology and Hydrodynamics is writing user interface plugins for QGIS and is wondering if these kind of user interfaces are the way to go in the future. If so, we should be able to co-create with QGIS and influence their technical roadmap.</p>"
```

File: radar.csv [hunk 3]
```diff
@@ -45,17 +46,20 @@ Gitlab on premise,Adopt,platforms,FALSE,"<p><strong><a href=""https://about.gitl
...
-Argo,Assess,platforms,TRUE,"<p><strong><a href=""https://argoproj.github.io/"">Argo</a></strong> is an open-source container-native workflow engine for orchestrating parallel jobs on Kubernetes.</p>"
-Kubernetes,Assess,platforms,TRUE,"<p><strong><a href=""https://kubernetes.io/"">Kubernetes</a></strong> is an open-source container-orchestration system for automating computer application deployment, scaling, and management.</p>"
+Dependabot,Trial,platforms,TRUE,"<p><strong><a href=""https://dependabot.com/"">Dependabot</a></strong> is a tool that automates dependency updates. It checks for updates to your dependencies every day and opens a pull request when it finds an update. You can configure Dependabot to open pull requests immediately or after a certain number of days. There are also vulnerability checks in place.</p><p>Motivation: Dependabot comes for free with our Github subscription. It is useful, but does not work for all ecosystems and only works for Github repositories. Still it is useful to enable it where possible.</p><p>Question: Can we use custom repositories for update checking?</p>"
```
Comment (joelvdberg) [inline radar.csv:55]: I had a hard time recalling what was meant with "Can we use custom repositories for update checking". Maybe this needs to made clearer:

```suggestion
Dependabot,Trial,platforms,TRUE,"<p><strong><a href=""https://dependabot.com/"">Dependabot</a></strong> is a tool that automates dependency updates. It checks for updates to your dependencies every day and opens a pull request when it finds an update. You can configure Dependabot to open pull requests immediately or after a certain number of days. There are also vulnerability checks in place.</p><p>Motivation: Dependabot comes for free with our Github subscription. It is useful, but does not work for all ecosystems and only works for Github repositories. Still it is useful to enable it where possible.</p><p>Question: Can dependabot check for updated packages in a privately hosted package repository?</p>"
```

File: radar.csv [hunk 4]
```diff
@@ -13,25 +13,26 @@ SpecFlow,Trial,languages-and-frameworks,FALSE,"<p><strong><a href=""https://spec
...
+Rust,Assess,languages-and-frameworks,TRUE,"<p><strong><a href=""https://www.rust-lang.org/"">Rust</a></strong> is a systems programming language that runs blazingly fast, prevents segfaults, and guarantees thread safety. Rust is a multi-paradigm language that is designed for performance and safety. It is particularly well-suited to writing low-level code, such as operating systems, device drivers, and embedded systems. Rust is a language that is designed to be memory-safe and thread-safe, and to prevent null pointer dereferences, buffer overflows, and other common programming errors.</p><p>Motivation: Deltares would like to try out projects with this language.</p>"
```
Comment (joelvdberg) [inline radar.csv:19]: If "Deltares would like to try out projects with this language", then shouldn't Rust be in quadrant _Trial_?

**Review by joelvdberg — CHANGES_REQUESTED** [review]

File: radar.csv (hunk 4 above)
Comment (deltamarnix) [reply radar.csv:19]: Depends on how we see it, we currently don't have any trials going on with this language.

**Review by joelvdberg — APPROVED** [review]

**Pull request merged**